### DISCUSSION TITLE
Love Poem Doesn't Change after 1000 affection

### DISCUSSION THREAD

**Discussion by matt-fackler** [Monika-After-Story/MonikaModDev#9308] [general]
I saw on the guide page that after you reach 1000 affection, a couple of new things trigger including the secret.txt turning into a full blown love poem.  However, it never did change for me.  While this doesn't exactly sour my gameplay that bad I just thought I would report the bug.  secret.txt is still stuck as the original text file with the original message.  All the other 1000 affection stuff triggered but not this.  Not sure what's up.

**Comment by Booplicate** [comment]
Wiki is outdated, `secret.txt` won't be updated, you will get a brand new poem, the old stuff remains unchanged.

**Comment by matt-fackler** [comment]
I see.  I guess the bug is that I didn't get a new poem.  It never showed up.  Does it appear in a different place?

**Comment by Booplicate** [comment]
It's random, you'll get it eventually. It will appear in the same folder.
